f3-sidecar/ec_test.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/filecoin-project/go-f3/ec"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	EC  ec.Backend
+	ctx = context.Background()
+)
+
+func init() {
+	ec, err := NewForestEC("http://localhost:2345/rpc/v1")
```

Comment:
I discourage using `localhost` if `127.0.0.1` can be used. To my knowledge, it may decrease performance significantly in some technologies (it may or may not be the case here, but better safe than sorry).

Reply:
Fixed.